src/service/viewport-impl.js

```diff
@@ -191,6 +191,17 @@ export class Viewport {
   }
 
   /**
+   * Sets the body border-bottom to the specified value.
+   * @param {number} borderBottom
+   */
+  setBorderBottom(borderBottom){
+    onDocumentReady(this.win_.document, () => {
```

Comment:
I don't know about this. I think `whenDocumentReady` works, but `onDocumentReady` has been used in this way in `updatePaddingTop` function.